tsconfig.json

```diff
@@ -25,35 +25,31 @@
     // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
 
     /* Modules */
-    "module": "commonjs",                                /* Specify what module code is generated. */
+    "module": "ESNext",                                  /* Specify what module code is generated. */
     // "rootDir": "./",                                  /* Specify the root folder within your source files. */
-    // "moduleResolution": "node",                       /* Specify how TypeScript looks up a file from a given module specifier. */
+    "moduleResolution": "node",                          /* Specify how TypeScript looks up a file from a given module specifier. */
```

Comment:
This is not part of vanilla ESM, but it is very common to have this set to `node` even for vanilla web projects, using import maps as needed, because this is the default and simplest way in which type definitions flow into code in the ESM format (TS automatically grabs types from node_modules, which is the most common way people currently install libraries).

This covers non-relative bare module specifiers in our tests (f.e. `import ... from 'some-lib'`), while code inside of `src/` does not actually have any such library imports. If code in `src/` did have library imports, I would have needed to add an `importmap` to my example at https://github.com/trusktr/prism-v2-test.

<img width="273" alt="Screenshot 2023-07-05 at 5 35 39 PM" src="https://github.com/PrismJS/prism/assets/297678/5a891cfb-d089-41b6-8bd5-0299a4fba81e">